docs/CaliptraHardwareSpecification.md

```diff
@@ -1493,6 +1493,10 @@ After programming the key vault read control, FW needs to query the associated k
 
 Similarly, after programming the key vault write control and initiating the cryptographic function that generates the key to be written, FW needs to query the associated key vault write status to confirm that the requested key was generated and written successfully.
 
+When a key is read from the keyvault, the API register is locked and any result generated from the cryptographic block is not readable by firmware. The digest can only be sent to the keyvault by appropriately programming the keyvault write controls. After the cryptographic block completes its operation, the lock is cleared and the key is cleared from the API registers.
```

Comment:
recommend adding a space between `key` and `vault` in this section. The spec isn't consistent about this term throughout but we tend to use lowercase with a space when using in plaintext.

Reply:
Noted - if I need to submit another commit here I'll make this update throughout the doc.